pkgs/by-name/la/lan-mouse/package.nix

```diff
@@ -1,30 +1,37 @@
-{ stdenv
-, rustPlatform
-, fetchFromGitHub
-, lib
-, darwin
-, glib
-, gtk4
-, libadwaita
-, libX11
-, libXtst
-, pkg-config
-, wrapGAppsHook4
+{
+  stdenv,
+  rustPlatform,
+  fetchFromGitHub,
+  lib,
+  glib,
+  gtk4,
+  libadwaita,
+  libX11,
+  libXtst,
+  pkg-config,
+  wrapGAppsHook4,
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "lan-mouse";
-  version = "0.9.1";
+  version = "0.10.0";
 
   src = fetchFromGitHub {
     owner = "feschber";
     repo = "lan-mouse";
     rev = "v${version}";
-    hash = "sha256-BadpYZnZJcifhe916/X+OGvTQ4FQeTLnoy0gP/i5cLA=";
+    hash = "sha256-ofiNgJbmf35pfRvZB3ZmMkCJuM7yYgNL+Dd5mZZqyNk=";
+  };
+
+  prePatch = ''
```

Comment:
In case, someone applies a patch using an override/overlay that affects `build.rs`, we should only delete the file after patches have applied.

```suggestion
  postPatch = ''
```

Reply:
Hmm, I see. The scenario in my head was that someone might backport an upstream commit that also happens to touch `build.rs`.

Either way, I think this is a non-blocking review comment :D